libs/angular-webcomponents/src/lib/utils/webcomponent-router.utils.ts

```diff
@@ -0,0 +1,17 @@
+import { Route, UrlMatcher, UrlSegment, UrlSegmentGroup } from '@angular/router'
+
+export function match(prefix: string): UrlMatcher {
```

Comment:
From the view of the app, this is startsWith, right? Maybe we should rename the function, even if the function takes the part handled by the shell into account